app/models/auth_sources/auth_source_ldap.rb

```diff
@@ -130,6 +131,23 @@ def users_in_group(name)
     ldap_con.user_list(name)
   end
 
+  def test_connection
+    result = {}
+    begin
+      Timeout::timeout(60) do
+        self.ldap_con.valid_user?("")
```

Comment:
No reply to my last comment?  Please submit a PR to theforeman/ldap_fluff to improve the test method if it isn't sufficient.  It'll allow us to easily extend tests in the future too, e.g. existence of base DNs and other useful bits.